### DISCUSSION TITLE
PLEASE LOAD SVG FROM URL

### DISCUSSION THREAD

**Discussion by raulidavid** [QuestPDF/QuestPDF#851] [Ideas]
PLEASE LOAD SVG FROM URL

**Comment by MarcinZiabek** [comment]
Thank you for reaching out 😄 

As the QuestPDF library focuses on document generation, it should not perform HTTP requests, primarily for performance and security reasons. 

I suggest downloading the SVG image in your code (likely by creating a helper method) and then providing already downloaded SVG content to the QuestPDF library.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
